### ISSUE TITLE
[Revise subclass] mucolipidosis type III gamma (MONDO_0009652) be at same level as mucolipidosis type III (MONDO_0018931)

### ISSUE THREAD

**Issue by lcdaugherty** [monarch-initiative/mondo#1787]
**Suggested revision and reasons**
Should mucolipidosis type III gamma  (MONDO_0009652) be at the same level as mucolipidosis type III (MONDO_0018931); both being subtypes of mucolipidosis (MONDO_0019248)?

**current**

<img width="437" alt="mucolipidosis MONDO_0019248 tree" src="https://user-images.githubusercontent.com/14925172/87799251-e98ec280-c844-11ea-9eb2-a0653fc623cf.png">

**proposed**
mucolipidosis (MONDO_0019248)
 -mucolipidosis type II MONDO_0009650 syn : mucolipidosis type II alpha/beta (GNPTAB)
 -mucolipidosis type III MONDO_0018931 syn : mucolipidosis type III alpha/beta  (GNPTAB)
 -mucolipidosis type III gamma MONDO_0009652 (GNPTG)
 -mucolipidosis type IV MONDO_0009653 (MCOLN1)

**Comment by nicolevasilevsky** [comment]
![image](https://user-images.githubusercontent.com/6722114/87836893-9158c000-c846-11ea-9283-b6d1033f3bd6.png)

If I remove the assertion of 'mucolipidosis type III' as a parent, MONDO_0009652 mucolipidosis type III gamma is still inferred to be a subclass of 'mucolipidosis type III' because of the logical definition.

I think the logical definition is accurate, so I think this hierarchy is correct.

Is it okay with you if we leave it as is?

Thanks!

**Comment by lcdaugherty** [comment]
Sorry @nicolevasilevsky  I am still have problems resolving this entry. 
The orphanet terms have a child/parent relationship, but are both exactmatch in
mucolipidosis type III MONDO:0018931 :
exactMatch Orphanet_577 Mucolipidosis type III
exactMatch Orphanet_423461 Mucolipidosis type III alpha/beta

**Comment by nicolevasilevsky** [comment]
@lcdaugherty no need to apologize!

Looks like we obsoleted 'pseudo-Hurler polydystrophy', which was equivalent to Orphanet_423461 Mucolipidosis type III alpha/beta, and merged it with mucolipidosis type III MONDO:0018931.

I revised the xref to Orphanet_423461 on mucolipidosis type III to be MONDO:obsoleteEquivalent, which is used for cases where we have exact 1:1 matches between an obsolete in Mondo and a live class in the source ontology. We want to avoid making an equivalence axiom (MONDO:equivalentTo) here.
(see editor docs: https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/)
